### ISSUE TITLE
alt text generation is blocking

### ISSUE THREAD

**Issue by cooperbarth** [InclusiveTechNU/bri11iant#23]
Generating alt text for `<img>` tags blocks other tags. Desired behavior should be:
- Default alt text is shown and all other tags are calculated
- Another message is sent once image classification finishes to update the description

**Comment by cooperbarth** [comment]
https://github.com/InclusiveTechNU/bri11iant/commit/7fd4f19b3ddc75f804d1211c5c00b6e8d44bf99c
